breathecode/events/tests/urls/tests_academy_eventype_slug.py

```diff
@@ -38,6 +38,21 @@ def get_serializer(event_type, academy=None, city=None, data={}):
     }
 
 
+def get_put_serializer(event_type, data={}):
```

Comment:
That name ain't right because you is not getting a put serializer, you is simulating a put serializer